### PR TITLE
Add customisation required for dark mode

### PR DESCRIPTION
**Pull request by timja**
image

### CODE REVIEW

File: core/src/main/resources/lib/layout/breadcrumbs.css [hunk 1]
```diff
@@ -51,6 +51,7 @@
 
 #breadcrumbs LI:hover, A.breadcrumbBarAnchor.mouseIsOverMenuSelector {
   background-color: #e0e4dc;
+  background-color: var(--breadcrumbs-item-bg-color--hover);
```
Comment (oleg-nenashev) [inline core/src/main/resources/lib/layout/breadcrumbs.css:54]: Do we need a double dash in `--hover`?

Comment (timja) [reply core/src/main/resources/lib/layout/breadcrumbs.css:54]: the other variables have them to indicate that they are an altered state from the default variable

https://github.com/jenkinsci/jenkins/blob/master/war/src/main/less/abstracts/colors.less#L117

**Review by EstherAF — APPROVED** [review]
looks good!

**Comment by timja** [comment]
🚢 ?

**Pull request merged**